<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="DataFrameView">
       <property name="title">
        <string>  Data Frame View</string>
       </property>
       <widget class="QTableWidget" name="DataViewTable">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>661</width>
          <height>531</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>451</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="RawDataLoad">
       <property name="title">
        <string>  Raw Data Load</string>
       </property>
       <widget class="QLabel" name="select_path_print">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="text">
         <string>Select file path</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="Path_btn">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>30</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Path</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Path_edit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Input data file path</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="Load">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>70</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="Header_int">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>110</y>
          <width>50</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="Header">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Header Row</string>
        </property>
       </widget>
       <widget class="QLabel" name="IndexColumn">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>110</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Index column</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Index_col">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>110</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string> Select Series ( Feature )</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>280</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string> Transform Series </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>450</y>
      <width>451</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string> Transform Series </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
